tiatoolbox/utils/misc.py

```diff
@@ -1212,9 +1216,16 @@ def patch_pred_store(
             conversion of annotations saved at non-baseline resolution to baseline.
             Should be model_mpp/slide_mpp.
         class_dict (dict): Optional dictionary mapping class indices to class names.
+        save_dir (str or pathlib.Path): Optional Output directory to save the Annotation
```

Comment:
```suggestion
        save_dir (str or Path): Optional Output directory to save the Annotation
```
It's imported as `Path`